### ISSUE TITLE
`/post` replay feature assumes JSON content

### ISSUE THREAD

**Issue by simonbasle** [gaul/java-httpbin#7]
performing eg. a form-encoded POST on the endpoint results in an exception:
```java
org.json.JSONException: A JSONObject text must begin with '{' at 1 [character 2 line 1]
```
From the perspective of my test it would also appear that the exception's translation to a 500 status somehow isn't visible to the client, which receives a 200 OK instead.
See this capture summary of the interaction from wireshark:

```
POST /post HTTP/1.1
user-agent: ReactorNetty/dev
host: localhost:61809
accept: */*
content-type: multipart/form-data; boundary=1cf61a8f8e6a35de
transfer-encoding: chunked

--1cf61a8f8e6a35de
content-disposition: form-data; name="test"
content-length: 95
content-type: application/octet-stream
content-transfer-encoding: binary

This is an UTF-8 file that is smaller than 1024 bytes.
It contains accents like é.
End of File
--1cf61a8f8e6a35de
content-disposition: form-data; name="attr1"
content-length: 5
content-type: text/plain; charset=UTF-8

attr2
--1cf61a8f8e6a35de
content-disposition: form-data; name="test2"
content-length: 0
content-type: application/octet-stream
content-transfer-encoding: binary


--1cf61a8f8e6a35de--
```
and response
```
HTTP/1.1 200 OK
Date: Thu, 16 May 2019 17:56:12 GMT
Content-Length: 0
Server: Jetty(9.2.12.v20150709)
```

**Comment by gaul** [comment]
I fixed the 500 status handling and deployed a new snapshot.  httpbin.org only documents a JSON response type; does it honor different request content types?  If so could you submit a pull request to add this?

**Comment by simonbasle** [comment]
the `/post` endpoint of httpbin.org indeed _accepts_ incoming `content-type: multipart/form-data`, but _responds_ in JSON.

**Comment by gaul** [comment]
@simonbasle Could you submit a pull request to accept different request content types?

**Comment by simonbasle** [comment]
I currently don't have much time to dedicate to this, unfortunately, as we have moved on with a different testing solution

**Comment by gaul** [comment]
@simonbasle I understand that you have moved on to another solution but could you test the latest snapshot since I only lightly tested the suggested feature?  I can cut a new release afterwards.

**Comment by simonbasle** [comment]
it seems to do it for `multipart/form-data` 👍

**Comment by gaul** [comment]
@simonbasle Thanks for your help!  I released 1.2.0.
